platform/core/src/studies/retrieveStudyMetadata.js

```diff
@@ -33,18 +40,75 @@ export function retrieveStudyMetadata(server, StudyInstanceUID, filters) {
   }
 
   // Create a promise to handle the data retrieval
-  const promise = new Promise((resolve, reject) => {
-    RetrieveMetadata(server, StudyInstanceUID, filters).then(function(data) {
-      resolve(data);
-    }, reject);
-  });
+  let promise;
+
+  if (
+    filters &&
+    filters.seriesInstanceUID &&
+    seperateSeriesInstanceUIDFilters
+  ) {
+    promise = _seperateSeriesRequestToAggregatePromise(
+      server,
+      StudyInstanceUID,
+      filters
+    );
+  } else {
+    promise = new Promise((resolve, reject) => {
```

Comment:
what is the point of this new promise? isn't it the same as promise = RetrieveMetadata(server, StudyInstanceUID, filters)?

Reply:
This is just what was in the previous code. Hmm you are right, weird. Removed and tested, all works fine.